app/src/main/java/com/zulip/android/util/Constants.java

```diff
@@ -13,5 +13,6 @@
     //Default maximum time limit for editing message(Same as server)
     public final static int DEFAULT_MAXIMUM_CONTENT_EDIT_LIMIT = 600;
     public final static boolean DEFAULT_EDITING_ALLOWED = true;
+    public static String serverUrl = null;
```

Comment:
Here https://github.com/zulip/zulip-android/pull/363#discussion_r99458136 I asked for the String tag you are passing, not the URL variable!  (text in the bold italics) 
String url = getIntent().getStringExtra("**_server_url_**");